### PR TITLE
feat: remove cookie length check

### PR DESCRIPTION
**Pull request by dead-horse**
##### Checklist

- [x] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

File: app/middleware/session.js [hunk 1]
```diff
@@ -1,27 +1,3 @@
...
-      if (!setCookie) {
-        return;
-      }
-      for (const val of setCookie) {
-        // http://browsercookielimits.squawky.net/
-        if (val.indexOf(cookieSessionMatchKey) === 0 && val.length > 4093) {
-          const err = new Error(`Max cookie limit is 4093, but got ${val.length}`);
-          err.name = 'CookieLimitExceedError';
-          err.cookie = val;
-          this.coreLogger.error(err);
```
Comment (fengmk2) [inline app/middleware/session.js]: https://github.com/eggjs/egg-cookies/pull/4/files#r102279854 改造后，这个日志没法打印了

**Comment by popomore** [comment]
这个等 egg 下个版本发了再合？

**Comment by dead-horse** [comment]
好

**Comment by popomore** [comment]
升级到 rc 版？

**Comment by dead-horse** [comment]
好了

**Comment by popomore** [comment]
这个要发 bk 吧，等 koa 改完一起发

**Pull request merged**